### ISSUE TITLE
The plugin is not compatible with Webpack 5

### ISSUE THREAD

**Issue by Pierre-Do** [honeybadger-io/honeybadger-webpack#288]
## What are the steps to reproduce this issue?

1. Configure the plugin as per documentation
2. Migrate from Webpac 4 to Webpack 5

## What happens?

Nothing, source-maps are not being uploaded

## What were you expecting to happen?

I'm expecting the source-maps to be uploaded, similar to Webpack 4.

## Any logs, error output, etc?

None

## Any other comments?

I believe this PR (https://github.com/thredup/rollbar-sourcemap-webpack-plugin/pull/327) needs to be backported to this plugin.

## What versions are you using?
**Operating System:** GNU/Linux (both locally and on Jenkins)
**Package Version:** 1.2.0
**Browser Version:** Irrelevant

**Comment by joshuap** [comment]
@Pierre-Do thanks for the report!
@ParamagicDev can you verify this when you get the chance?

**Comment by Pierre-Do** [comment]
Thanks @joshuap. I'm more than happy to start a PR myself if it helps! https://github.com/honeybadger-io/honeybadger-webpack/pull/289/files

**Comment by Pierre-Do** [comment]
Hey @ParamagicDev , I'm playing with this Webpack 5 compatibility (heavily inspired by the Rollbar plugin!), and I was able to upload locally some source-maps with Webpack 5 using these modification:

https://github.com/Pierre-Do/honeybadger-webpack/pull/1

This contains the fix for `auxiliaryFiles`, but also a change that reads the content of the source maps and files from the disk rather than from `compilation.assets[sourceMap].source()`.

That's a very naive PR that seems to work with my configuration, but it might give us more insights on what's needed for Webpack 5.

**Comment by joshuap** [comment]
I've published 1.3.0-beta.0 with this fix. @Pierre-Do can you try it out?

https://www.npmjs.com/package/@honeybadger-io/webpack/v/1.3.0-beta.0

**Comment by Pierre-Do** [comment]
Thanks @joshuap and @ParamagicDev for tackling this issue so quickly!

I went a bit ahead and released a package from a fork that went in production at QoQa this morning (this commit: https://github.com/qoqa/honeybadger-webpack/commit/914bc09ca0445904a6bfe5f9e44d273ce5a8a11b). It went well, although I'm intending to switch back to the official release asap.

I'll try https://www.npmjs.com/package/@honeybadger-io/webpack/v/1.3.0-beta.0 tomorrow or the day after. I'll let you know!

**Comment by Pierre-Do** [comment]
I've run the following tests: on a docker image based on `ruby-2.7.2` with Node 14, I've compiled my App with Webpack ` 5.24.4`. This yielded **210** source-maps to be uploaded to our Honeybadger project for our test environment. 

The build succeeded and the source-maps are being uploaded.

![image](https://user-images.githubusercontent.com/1904504/112105743-10a84b80-8bad-11eb-8b0e-79d04b568082.png)

I've done it from my local machine and through our standard Jenkins build.

I do not know if the source-maps are being properly instrumented by Honeybadger, but I'd assume so. I'll push the package in production during the week.

Reasonable I believe we can say that the issue with Webpack 5 is closed! :heavy_check_mark:  :tada: 

Just a side point if I may. We've noticed that since we're using the latest version of `@honeybadger-io/webpack` we had a couple of Jenkins jobs hanging just after the source-maps are being uploaded and the end of the Webpack build process. These kind of issues are very complex to debug, as they can be caused by any number of things in the build process, from Jenkins to Webpack to shear bad luck. As I'm not able to reproduce this behaviour locally, I'm not able to diagnose the source of the issue (yet). It's very unlikely that this issue is caused by `@honeybadger-io/webpack`, but let's keep our eyes open in case other people have the same issue.

Thanks again for all your work at Honeybadger!
